### PR TITLE
added ZactuatorSomfy

### LINKED ISSUE #814: Add support for Somfy RTS remote control
**Is your feature request related to a problem? Please describe.**
I want to control my Somfy RTS devices such as Somfy Altus RTS via OpenMQTTGateway.
Somfy RTS requires 433.**42** Mhz transmitter, see also #812.

**Describe the solution you'd like**
I created the [Somfy_Remote_Lib](https://github.com/Legion2/Somfy_Remote_Lib) which can emulate a Somfy RTS remote control. It should work on Arduino, ESP32 and ESP8266 boards.
As part of this feature request a Somfy RTS Gateway should be created. The Gateway should be able to control Somfy RTS devices in response to MQTT messages.

**Describe alternatives you've considered**
I tried adding support to ESPiLight but then I created the standalone lib, see https://github.com/puuu/ESPiLight/issues/56

**Additional context**
I can test on ESP32 and with the CC1101 module. This would also make #812 obsolete

### PR DESCRIPTION
**Pull request by Legion2**
Add support for Somfy RTS remote control using https://github.com/Legion2/Somfy_Remote_Lib.
close #814

I can only test with ESP32 and CC1101 module. I also created a platformio env for esp8266.

TODO:
- [x] Documentation
- [x] Licence Headers

### CODE REVIEW

**Comment by Legion2** [comment]
@1technophile What license header should be used for the new files?

**Comment by 1technophile** [comment]
Good question,
I would say this:

```
/*  
  OpenMQTTGateway  - ESP8266 or Arduino program for home automation 
   Act as a wifi or ethernet gateway between your 433mhz/infrared IR signal  and a MQTT broker 
   Send and receiving command by MQTT
 
  This gateway enables to:
 - receive MQTT data from a topic and send SMS corresponding to the received MQTT data
 - publish MQTT data to a different topic related to received SMS
  
    Copyright (C) 2020  name(s) of the copyrighted

    This file is part of OpenMQTTGateway.
    
    OpenMQTTGateway is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
    OpenMQTTGateway is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
*/
```

**Comment by Legion2** [comment]
>   This gateway enables to:
> - receive MQTT data from a topic and send SMS corresponding to the received MQTT data
> - publish MQTT data to a different topic related to received SMS

Should I change this?

**Comment by 1technophile** [comment]
Yes with a short description of the module.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: "14.x"
-      - name: Install vuepress
-        run: npm install -g vuepress
+      - name: Install build dependencies
+        run: npm install
```
Comment (1technophile) [inline .github/workflows/build.yml:32]: This modification should be on a separate PR I think

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -48,8 +48,8 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: "14.x"
-      - name: Install vuepress
-        run: npm install -g vuepress
+      - name: Install build dependencies
+        run: npm install
```
Comment (1technophile) [inline .github/workflows/release.yml:52]: same

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,4 @@
 .vscode
 *_env.ini
 **/.DS_Store
+node_modules
```
Comment (1technophile) [inline .gitignore:5]: same

File: docs/use/actuators.md [hunk 4]
```diff
@@ -74,3 +74,42 @@ After this calibration, if you set the `r` channel to 0.0, it will be remapped t
 Also, the gamma curve for this channel will be set to 2.5.
 This means that input values are raised to the power 2.5 internally.
 This can be used to improve the linearity of inputs.
+
+## Somfy RTS
+This actor allows to control Somfy RTS devices.
```
Comment (1technophile) [inline docs/use/actuators.md:79]: This actuator

File: docs/use/actuators.md [hunk 5]
```diff
@@ -74,3 +74,42 @@ After this calibration, if you set the `r` channel to 0.0, it will be remapped t
...
+
+### Setup
+Before the module can be used virtual Somfy RTS remotes must be created.
```
Comment (1technophile) [inline docs/use/actuators.md:82]: Before the module can be used virtual"," Somfy RTS remotes must be created.

File: package.json [hunk 6]
```diff
@@ -2,5 +2,8 @@
   "scripts": {
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs"
+  },
+  "devDependencies": {
+    "vuepress": "^1.7.1"
```
Comment (1technophile) [inline package.json:7]: This modification should be on a separate PR I think

File: package-lock.json [hunk 7]
```diff
@@ -2,180 +2,11027 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
+    "@babel/code-frame": {
```
Comment (1technophile) [inline package-lock.json:5]: This modification should be on a separate PR I think

**Review by 1technophile — COMMENTED** [review]
Thanks for the PR!

**Review by 1technophile — APPROVED** [review]
